asciimatics/screen.py

```diff
@@ -2550,7 +2552,9 @@ def get_event(self):
 
                     # Handle a genuine key press.
                     logger.debug("Returning key: %x", key)
-                    if key in self._KEY_MAP:
+                    if is_utf_char:
```

Comment:
Thinking about it, it's possible to trigger this off of the value of `_bytes_to_return` if we want to avoid an extra internal property.  I don't mind either way.

Reply:
I agree - `_bytes_to_return` is checked before returning a `KeyboardEvent` and when we find out that the key is a utf-8 character, we're reseting this value, so that for the next character this condition will be false unless we meet another utf-8 character.